docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,195 @@
 
-The displayed layout is preserved as much as possible, with new components
-appearing along a diagonal line. 
+Units
+    *Units* is the term used for any geometric shape in the flowsheet that is connected by lines.
+    In addition to different shapes for the different IDAES unit models, there are also designated
+    shapes for inlets and outlets. All units can be moved by clicking and dragging them.
+    If you right-click on a unit, it will rotate 90 degrees.
 
-.. note::
-    This feature is still under development. 
-    Several types of changes to the flowsheet currently cause the entire user-
-    modified layout to be lost. Consider saving the layout often (see below).
+Lines
+    The lines connecting units can be manipulated by clicking and dragging.
+    If you double-click on a line, you will create a new segment that can be used for routing the line
+    around objects. You can eliminate a segment by clicking on the dot that appears as you hover over
+    the line.
 
-.. image:: ../../_images/modelvis/new_unit_model_layout.png
+Annotations
+    Both the units and the arcs have associated values that can be shown. See the
+    :ref:`View:Labels <ifv-action-view>` action.
 
-6. Save the displayed layout using the save button on the visualization page. 
-   This writes the visualization to a file in the user's home directory under 
-   `.idaes/viz` using the model name provided to `visualize()`. 
-   In this example the filename would be `model_name.viz`.
+.. figure:: ../../_images/ifv_screenshot_diagramcontrols.png
+    :width: 800
 
-.. _streamlabels:
+    Screenshot of the diagram controls area of the IFV UI
 
-Stream Labels
--------------
+View actions
+    * Labels - Toggle view of the annotations, or labels
+    * Grid - Toggle a background "grid"
+    * Zoom - Zoom the view of the flowsheet within the canvas. This is a label for a set of related options.
+    *  Zoom in - Zoom in by 25%
+    *  Zoom out - Zoom out by 25%
+    *  Reset - Reset zoom to 100%
+    * Canvas size - Change the size of the "canvas" on which the flowsheet is drawn. This lets you adapt
+      the IFV for different display (screen) sizes.
 
-The initial layout loads with the stream labels hidden. Show or hide all of 
-the stream labels by clicking the button with the speech bubbles, 
-on the toolbar.
+.. figure:: ../../_images/ifv_screenshot_streamtable.png
+    :width: 800
 
-Show or hide an individual label by right clicking on the stream or its label.
+    Screenshot of the stream table area of the IFV UI
 
-.. _miscfeatures:
+Reference
+----------
 
-Misc. Features
---------------
+There are two ways to invoke the `visualize` functionality, which in the end do the same thing and
+have the same arguments.
 
-* Right click on an icon to rotate it by 90 degrees.
+1. Use the `visualize` method of a flowsheet
+2. Call the `visualize` function from the package `idaes.ui.fsvis`
+3. Call the same `visualize` function from the module `idaes.ui.fsvis.fsvis`
```

Comment:
You mentioned two ways to invoke the `visualize` functionality but you list three.